### ISSUE TITLE
:Behavior:Win32/DefenseEvasion.A!ml  is what i get when trying to extract uevr injector 

### ISSUE THREAD

**Issue by clanphear** [praydog/UEVR#8]
:Behavior:Win32/DefenseEvasion.A!ml  is what i get when trying to extract uevr injector ,  how do i fix this its attched to injector?
 
finally got it working after hours of trouble shooting .  but now ive noticed my f?>:"ing  the b button is on wrong side and impossible to run and crouch at the same time. looking for a way to swap like vorpx but couldnt find the tab if there is one. how do i fix

**Comment by eric-rolph** [comment]
Detected:  Trojan:Script/Wacatac.B!ml
Affected item: file: C:\Users\\Downloads\UEVR.zip

**Comment by DannyDelusion** [comment]
DefenseEvasion.A!ml is a false positive.

the **!ml** stands for "_Machine Learning_" as in, it's AI flagging it as potentially malicious because of certain things it does, so without looking into it, it'll be injecting or modifying file structures etc.

tl;dr i'm sure it's fine, it's a pretty common false positive.

**Comment by dannietjoh** [comment]
Disabled defender, unpacked, scanned again, nothing found. I guess the zip has to be submitted to Microsoft as false positive.

**Comment by qant** [comment]
chrome prevent to download, becase is suspicious  ) Rai Pal exe was removed by windows defender... nice start... i wonder why there is nothing about this in read me and from testers...

**Comment by DannyDelusion** [comment]
> chrome prevent to download, becase is suspicious ) Rai Pal exe was removed by windows defender... nice start... i wonder why there is nothing about this in read me and from testers...

Rai Pal has already been reported to MS as false positive.
balls in their court now.

**Comment by keyandishou** [comment]
Thank you guys, I encountered the same false positive.
